bot/constants.py

```diff
@@ -259,6 +254,7 @@ class Roles(NamedTuple):
     rockstars = 458226413825294336
     core_developers = 587606783669829632
     events_lead = 778361735739998228
+    everyone = 267624335836053506
```

Comment:
I'd prefer if you call this `everyone_role` to better match the python-bot constants. Could you also remove the developer role from the list?